packages/medusa/src/services/region.ts

```diff
@@ -533,7 +533,28 @@ class RegionService extends TransactionBaseService {
       this.regionRepository_
     )
 
+    let q: string | undefined
+
+    if (selector.q) {
+      q = selector.q
+      delete selector.q
+    }
+
     const query = buildQuery(selector, config)
+
+    if (q) {
+      const where = query.where as FindOptionsWhere<Region>
+
+      delete where.name
```

Comment:
@kasperkristensen, only included the name for now. Let me know if this is sufficient.

Reply:
I think that is fine, the only other option I think would make sense was to allow search by country, like how we have search product variants by product title: https://github.com/medusajs/medusa/blob/f29948a6a8189d9662ad28ba6d0a41c5bf2d4df8/packages/medusa/src/services/product-variant.ts#L923-L949

Would be cool to be able to search "denmark" and then "Europe" pops up, but I am good with leaving that for a future iteration 😄